common.py

```diff
@@ -0,0 +1,111 @@
+def get_month_year_from_string(string):
+    """
+    return month and year from combined string
+
+    month and year should be numbers and separated with /
+    Examples:
+        input string: 12/2006
+            output: 12, 2006
+        input string: jan/2006
+            output: None, None
+    """
+
+    month, year = string.split('/')
+    try:
+        month = int(month)
+        year = int(year)
+    except Exception:
+        print("Invalid input")
```

Comment:
add more information to this print statement, include reason why the input is invalid.